tests/testMaskedImage.py

```diff
@@ -120,9 +120,10 @@ def tearDown(self):
 
     def testArrays(self):
         image, mask, variance = self.mimage.getArrays()
-        self.assert_((self.mimage.getImage().getArray() == image).all())
-        self.assert_((self.mimage.getMask().getArray() == mask).all())
-        self.assert_((self.mimage.getVariance().getArray() == variance).all())
+        self.assertListEqual(self.mimage.getImage().getArray().flatten().tolist(), image.flatten().tolist())
```

Comment:
I think I did this before I knew about the assertFloatsEqual method, so I added the .flatten().tolist(). I'll update this to assertFloatsEqual, which will be much more clear.